Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -149,7 +179,7 @@ static QualifiedName saveAttr()
 }
 
 template <typename ElementType>
-static Ref<ElementType> createContainedElement(HTMLElement& container, const AtomString& id, String textContent)
+static Ref<ElementType> createContainedElement(Element& container, const AtomString& id, String textContent = { })
```

Comment:
`String&&` or `const String&`

Reply:
`String&&` produces identical code (at the cost of having to explicitly create a temporary in some callers -- that's why I initially preferred `String`), so I'll go with that.

Just curious: What is the reason to prefer passing by reference? Is this project-wide unspoken knowledge? It may be good to add it to https://webkit.org/code-style-guidelines/